Storage/SQL/LoginsSchema.swift

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+import XCGLogger
+
+let TableLoginsMirror = "loginsM"
+let TableLoginsLocal = "loginsL"
+let IndexLoginsOverrideHostname = "idx_loginsM_is_overridden_hostname"
+let IndexLoginsDeletedHostname = "idx_loginsL_is_deleted_hostname"
+
+private let AllTables: [String] = [
+    TableLoginsMirror,
+    TableLoginsLocal
+]
+
+private let log = Logger.syncLogger
+
+open class LoginsSchema: Schema {
```

Comment:
I hadn't considered breaking blame here. However, as someone who has recently stepped into this code, this was extremely unclear having this schema exist inside of another source file. IMO, since this file is relatively short (~100 lines), I feel the pros outweigh the cons here.